### DISCUSSION TITLE
Cannot access nested pages

### DISCUSSION THREAD

**Discussion by mattyclarkson** [community/community#21898] [Other Feature Feedback, Questions, & Ideas]
<p>I have published my pages to <a href="https://ef-carbon.github.io/react-native-async-button/" rel="nofollow noopener">https://ef-carbon.github.io/react-native-async-button/</a> from <a href="https://github.com/ef-carbon/react-native-async-button/tree/gh-pages" rel="nofollow noopener">https://github.com/ef-carbon/react-native-async-button/tree/gh-pages</a>. However, I cannot access any nested pages such as <a href="https://github.com/ef-carbon/react-native-async-button/blob/gh-pages/modules/_asyncbutton_component_.html" rel="nofollow noopener">https://github.com/ef-carbon/react-native-async-button/blob/gh-pages/modules/_asyncbutton_component_.html</a>, which should be exposed at <a href="https://ef-carbon.github.io/react-native-async-button/modules/_asyncbutton_component_.html" rel="nofollow noopener">https://ef-carbon.github.io/react-native-async-button/modules/_asyncbutton_component_.html</a>. Do I have to configure something to make this be available through GitLab pages?</p>

**Comment by birdcar** [comment, marked as answer]
<p>Hi <a class="mention" href="/mattyclarkson">@mattyclarkson</a>!</p>
<p>GitHub Pages uses Jekyll to build all sites by default, and as a result, files with a leading underscore are ignored during the build process by default. More information can be found here:</p>
<p><a href="https://help.github.com/articles/files-that-start-with-an-underscore-are-missing/" rel="nofollow noopener">https://help.github.com/articles/files-that-start-with-an-underscore-are-missing/</a></p>
<p>Since it doesn’t appear that you’re using Jekyll to build your site, the simplest fix for this would be to add an empty “.nojekyll” file to the root of your gh-pages branch. This will disable Jekyll’s build process and publish your site as is.</p>
